test/gui/Makefile

```
@@ -1,3 +1,6 @@
+# Automatically detect the latest Squish version directory
+SQUISH_DIR := $(shell ls -d $$HOME/squish-for-qt-* 2>/dev/null | sort -V | tail -1)
```

It might not work for all setups. squish installation path can be different from this.
```suggestion
SQUISH_DIR := $(shell ls -d $$HOME/squish-for-qt-* 2>/dev/null | sort -V | tail -1)
```